src/rrt/Tree.hpp

```diff
@@ -52,7 +68,10 @@ class Node {
      */
     const T& state() const { return _state; }
 
+    double* coordinates() { return _vec.data(); }
```

Comment:
Another reason is that you don't get bounds checking with `double*` which makes it fun to debug when someone messes up